front/ui/storybook/stories/ui-core/Dialog.stories.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from 'react';
+
+import { Button, ComboBox, Input, Dialog, TextArea } from '@osrd-project/ui-core';
+import { Search } from '@osrd-project/ui-icons';
+import { type Meta, type StoryObj } from '@storybook/react-vite';
+
+import '@osrd-project/ui-core/dist/theme.css';
+
+type StoryProps = {
+  btnLabel: string;
+  className?: string;
+  header: React.ReactNode;
+  body: React.ReactNode;
+  footer: (setOpen: (open: boolean) => void) => React.ReactNode;
+};
+const Wrapper = ({ btnLabel, className, header, body, footer }: StoryProps) => {
+  const [open, setOpen] = useState(true);
+  return (
+    <>
+      <Button onClick={() => setOpen(true)} label={btnLabel} />
+      {open && (
+        <Dialog className={className}>
+          {header}
+          {body}
+          {footer(setOpen)}
+        </Dialog>
+      )}
+    </>
+  );
+};
+
+const meta: Meta<typeof Wrapper> = {
+  component: Wrapper,
+  args: {},
+  title: 'Core/Dialog',
+
+  tags: ['autodocs'],
+};
+
+export default meta;
+
+type Story = StoryObj<typeof Wrapper>;
+
+export const FormDialog: Story = {
+  args: {
+    btnLabel: 'Dialog with form',
+    className: 'ambientB',
+    header: <h5>Edit local variant</h5>,
+    body: (
+      <form id="form" style={{ width: '406px', margin: '0 auto' }}>
+        <Input id="variant" label="Variant name" type="text" defaultValue="Variant 003004" />
+        <ComboBox
+          id="parent"
+          label="Parent train group"
+          value="Group DDDEEEFFF"
+          suggestions={['Group DDDEEEFFF', 'Group AAABBBCCC']}
+          getSuggestionLabel={(e) => e}
+          onSelectSuggestion={() => {}}
+          resetSuggestions={() => {}}
+        />
+        <TextArea
+          id="description"
+          label="Variant description"
+          value="Testing more trains upstream"
+        />
+      </form>
+    ),
+    footer: (setOpen) => (
+      <>
+        <Button variant="Cancel" label="Cancel" onClick={() => setOpen(false)} />
+        <Button type="submit" form="form" label="Edit variant" onClick={() => setOpen(false)} />
+      </>
+    ),
+  },
+};
+
+export const ConfirmDialog: Story = {
+  args: {
+    btnLabel: 'Confirm dialog',
+    className: 'ambientB',
+    header: <h5>Removing a local variant</h5>,
+    body: (
+      <>
+        <p>You&apos;re about to remove &lsquo;Variant 003004&rsquo; from this scenario.</p>
+        <p style={{ color: 'var(--color-error-60)', fontWeight: 'bold' }}>
```

Comment:
```suggestion
        <p>You&apos;re about to remove &ldquo;Variant 003004&rdquo; from this scenario.</p>
        <p style={{ color: 'var(--color-error-60)', fontWeight: '600' }}>
```
Just to match the mockup ^^

Reply:
Thanks @SarahBellaha 

To make my stories for the Dialog, I used the design of my feature, it's always better to use a real example.
And you have right for the Select & Textarea,  but : 
- I just found the select component (which is not part of "input" folder...). 
- I used the textare in ui-core/inputs, and it's a bug on it.  I just created this ticket #14383